lib/rails_response_dumper/runner.rb

```diff
@@ -55,7 +55,8 @@ def run_dumps
             dump = {
               request: {
                 method: request.method,
-                url: request.url
+                url: request.url,
+                body: request.body.string
```

Comment:
Any reason to not include trailing comma here?

Reply:
A trailing comma would violate the RuboCop rules. I'm open to changing the rules, but that is the only reason.

```
lib/rails_response_dumper/runner.rb:59:42: C: [Corrected] Style/TrailingCommaInHashLiteral: Avoid comma after the last item of a hash.
                body: request.body.string,
```